packages/gui/src/components/plotNFT/select/PlotNFTSelectPool.tsx

```diff
@@ -86,6 +87,9 @@ const PlotNFTSelectPool = forwardRef((props: Props, ref) => {
   } = props;
   const [loading, setLoading] = useState<boolean>(false);
   const { balance, loading: walletLoading } = useStandardWallet();
+  const { nfts } = usePlotNFTs();
+  const openDialog = useOpenDialog();
+  let createNFT;
```

Comment:
This should be moved into `handleSubmit` since it isn't used elsewhere. I would also initialize it to `true` to avoid the need to check for `undefined`.

Reply:
Moved createNFT into handle submit, others could not be moved.